### PR TITLE
Move the deployment to Heroku

### PR DESCRIPTION
**Pull request by jwm**
Move the deployment to Heroku

This makes changes to move our primary deployment to Heroku. We'll need to keep linode2.openhatch.org for the git/svn missions, since it's not feasible to keep the repo state on a Heroku dyno.

I wanted to open this for review and feedback, but we shouldn't merge this until we actually perform the cut, since there are other operational changes that need to happen along with this code - there's a checklist in #1590.

### CODE REVIEW

**Comment by willingc** [comment]
@jwm Yay! I adjusted the codecov tool settings so the next time you push to this PR it should not complain.

File: mysite/deployment_settings.py [hunk 1]
```diff
@@ -50,7 +48,10 @@
 # svn mission requires a subdomain svn-mission for cloudflare to
 # work properly
 SVN_REPO_URL_PREFIX = 'svn://svn-mission.openhatch.org/'
-GIT_REPO_URL_PREFIX = 'https://openhatch.org/git-mission-data/git/'
+GIT_REPO_URL_PREFIX = 'https://git-mission.openhatch.org/git-mission-data/git/'
```
Comment (willingc) [inline mysite/deployment_settings.py]: Has the `git-mission` subdomain been set up yet? Or is this still a TODO?

Comment (jwm) [inline mysite/deployment_settings.py]: That's still a TODO.

Comment (paulproteus) [inline mysite/deployment_settings.py]: I've just GPG emailed you the CloudFlare password, which should be all the
credentials you need to configure DNS. Let me know if you agree/disagree!

Comment (jwm) [inline mysite/deployment_settings.py]: Got them, agreed, that should be all I need - thanks!

**Comment by willingc** [comment]
@jwm :bouquet: :balloon: :tada: :penguin:

**Pull request merged**